gulpfile.js

```diff
@@ -120,6 +121,12 @@ gulp.task("compile:favicons", function() {
       .pipe(gulp.dest("dist/favicons"));
 });
 
+gulp.task('validate:example', 'validate example html files', function() {
+  return gulp.src('src/*.html')
+    .pipe(compileExample('./src/templates/', 'example.html'))
```

Comment:
The validate task doesn't perform a build or touch the output directory. It just tests that the code in "src" will produce valid amp. If we can get validate to run fast (~ 1s) then we could insert it into the build.